cli/command/run/cmd.go

```diff
@@ -97,9 +98,11 @@ func Command(cli *command.SauceCtlCli) *cobra.Command {
 	cmd.Flags().IntVar(&concurrency, "ccy", 1, "Concurrency specifies how many suites are run at the same time.")
 	cmd.Flags().StringVar(&tunnelID, "tunnel-id", "", "Sets the sauce-connect tunnel ID to be used for the run.")
 	cmd.Flags().StringVar(&tunnelParent, "tunnel-parent", "", "Sets the sauce-connect tunnel parent to be used for the run.")
+	cmd.Flags().StringVar(&runnerVersion, "runner-version", "", "Sets runner_version.")
```

Comment:
A better description of the flag would be good. Describing the flag with itself is not very descriptive ;)

```suggestion
	cmd.Flags().StringVar(&runnerVersion, "runner-version", "", "Overrides the automatically determined runner version.")
```

Reply:
nice! updated.